operatorcourier/validate.py

```diff
@@ -273,8 +273,11 @@ def _csv_spec_validation(self, spec, bundleData):
                     if 'version' in csvOwnedCrd:
                         if 'spec' in crd:
                             if 'versions' in crd['spec']:
-                                if csvOwnedCrd['version'] not in crd['spec']['versions']:
-                                    self._log_error('CSV.spec.crd.owned.version is'
+                                if csvOwnedCrd['version'] not in [
```

Comment:
This should also applied to line 284

Reply:
I don't think so:
crd['spec']['versions'] is a list of dict where the relevant value is defined by the name key, crd['spec']['version'] is instead a plain text value.

Reply:
Just checked again, you are right

Reply:
but you are right, there is another issue